e2e/fixtures/schemas/result.json

```
@@ -0,0 +1,194 @@
+                            "required": [
+                                "file_name",
+                                "similarity_id",
+                                "line",
+                                "issue_type",
+                                "search_key",
+                                "search_value",
+                                "expected_value",
+                                "actual_value",
+                                "value"
+                            ],
+                            "properties": {
+                                "file_name": {
+                                    "type": "string",
+                                    "minLength": 1
+                                },
+                                "similarity_id": {
+                                    "type": "string",
+                                    "pattern": "^[A-Fa-f0-9]{64}$"
+                                },
+                                "line": {
+                                    "type": "integer",
+                                    "minimum": 1
+                                },
+                                "issue_type": {
+                                    "type": "string",
+                                    "enum": ["IncorrectValue", "MissingAttribute", "RedundantAttribute"]
+                                },
+                                "search_key": {
+                                    "type": "string",
+                                    "minLength": 0
+                                },
+                                "search_value": {
+                                    "type": "string"
+                                },
+                                "expected_value": {
+                                    "type": "string",
+                                    "minLength": 0
+                                },
+                                "actual_value": {
+                                    "type": "string",
+                                    "minLength": 0
+                                },
+                                "value": {
+                                    "type": ["string", "null"]
+                                }
+                            }
+                        }
+                    },
+                    "category": {
+                        "type": "string",
+                        "enum": [
+                            "Availability", "Backup", "Best Practices", "Build Process",
+                            "Encryption", "Insecure Configurations", "Insecure Defaults",
+                            "Networking and Firewall", "Observability", "Resource Management",
+                            "Secret Management", "Supply-Chain", "Structure and Semantics"
+                        ]
+                    },
+                    "description": {
+                        "type": "string"
+                    }
```

I think we could add `minLength: 1`, since description **must** not be empty...
```suggestion
                    "description": {
                        "type": "string",
                        "minLength": 1
                    }
```